arch/arm/core/aarch32/cortex_m/mpu/arm_core_mpu.c

```diff
@@ -209,9 +209,8 @@ void z_arm_configure_dynamic_mpu_regions(struct k_thread *thread)
 	LOG_DBG("configure user thread %p's context", thread);
 	if (thread->arch.priv_stack_start) {
 		/* K_USER thread stack needs a region */
-		uint32_t base = (uint32_t)thread->stack_obj;
-		uint32_t size = thread->stack_info.size +
-			(thread->stack_info.start - base);
+		uint32_t base = (uint32_t)thread->stack_info.start;
```

Comment:
Yeap, this is now required, since stack_info.start and .size are, now, modified, when the thread enters user mode.
Should we add an ASSERT that thread->stack_obj equals stack_info.start?

Reply:
No I do not think we should do that. It would break things if, for example, we moved the privilege elevation stack within the object.